wallet/mixing.go

```diff
@@ -105,6 +105,7 @@ func (w *Wallet) MixOutput(ctx context.Context, dialTLS DialFunc, csppserver str
 SplitPoints:
 	for i = 0; i < len(splitPoints); i++ {
 		last := i == len(splitPoints)-1
+		mixValue = splitPoints[i]
```

Comment:
Prior to https://github.com/decred/dcrwallet/pull/1995/commits/d479a125eb66c956522832a764afbee9df4145e7, there was a `range` instead, which actually seems better here.  Any reason not to stick with `for i, mixValue = range splitPoints`?

Reply:
We need to see if we iterated past the last item in the array, `i` is declared before this loop and checked immediately after.